src/user/resource.js

```diff
@@ -108,6 +108,10 @@ angular
         method: "DELETE",
         url: endpoint + '/subscription'
       },
+      subscriptionAction: {
+        method: "POST",
+        url: endpoint + '/subscription/:subscriptionId/action',
```

Comment:
I think you need to add
```
params : {
    subscriptionId : '@'
}
```

this tells Angular that the subscriptionId param should be used in the URL and not passed as a query string.